cola/widgets/dag.py

```diff
@@ -586,6 +587,10 @@ def __init__(self, context, params, parent=None, settings=None):
         self.refresh_action = qtutils.add_action(
             self, N_('Refresh'), self.refresh, hotkeys.REFRESH)
 
+        self.revtext_gather_paths_action = qtutils.add_action_bool(
+            self, N_('Gather Paths in Log'), self.set_revtext_gather_paths,
```

Comment:
The user-visible label here might make more sense as simply `N_('Autocomplete file paths')` that way we don't mention `Log` or "gathering", which might not make sense in some contexts.

I'm kinda curious -- what was the original motivation for this change?  Is the autocomplete for paths getting too slow when working in large repos?  If so thank you for improving performance for folks that have large repos.